### PR TITLE
asadiqbal08-OLIVE-74 RTL: Checkbox icon overlaying Preview button when system is in RTL language 

### PR DESCRIPTION
**Pull request by asadiqbal08**
[TASK-OLIVE74](https://openedx.atlassian.net/browse/OLIVE-74)

Here is the [sandbox](https://studio-asadiqbal08.sandbox.edx.org/container/block-v1:uetX+CS212+2016_RR+type@vertical+block@e8e0f63e6149472ab5f459f89d6462dc?action=new) that is showing the desired behavior

### CODE REVIEW

File: done/static/css/done.css [hunk 1]
```diff
@@ -94,4 +94,34 @@
...
+body.rtl .done_onoffswitch-checkbox:checked + .done_onoffswitch-label .done_onoffswitch-switch {
+    right: 0px;
+}
+
+body.rtl .done_onoffswitch-inner {
+    margin-right: -100%;
+}
+
+body.rtl .done_onoffswitch-inner:before {
+    content: "Unmark";
```
Comment (andy-armstrong) [inline done/static/css/done.css]: There shouldn't be strings in the CSS as they won't get localized.

File: done/static/css/done.css [hunk 2]
```diff
@@ -94,4 +94,34 @@
...
+    padding-right: 10px;
+    background-color: #EEEEEE;
```
Comment (andy-armstrong) [inline done/static/css/done.css]: Why are there colors that are specified in RTL? I would expect the only difference in the RTL version to be related to left/right layout.

File: done/static/css/done.css [hunk 3]
```diff
@@ -94,4 +94,34 @@
 
 .done_nogrow {
     flex-grow: 0;
-}
\ No newline at end of file
+}
+
+/* Right to left (rtl) language css */
+
+body.rtl .done_onoffswitch-checkbox:checked + .done_onoffswitch-label .done_onoffswitch-inner {
```
Comment (andy-armstrong) [inline done/static/css/done.css]: Is it possible to use bi-app here the same way that we do in platform? Then you don't need to implement styling for right-to-left languages, as every function you use is automatically language aware. I haven't tried to do this in a separate XBlock repo, so maybe that isn't possible.

File: done/static/css/done.css [hunk 4]
```diff
@@ -94,4 +94,34 @@
...
+body.rtl .done_onoffswitch-inner:before {
+    content: "Unmark";
+    padding-right: 10px;
+    background-color: #EEEEEE;
+    color: #999999;
+    text-align: right;
+}
+
+body.rtl .done_onoffswitch-inner:after {
+    content: "Mark as complete";
```
Comment (andy-armstrong) [inline done/static/css/done.css]: No strings in the CSS.

File: done/static/css/done.css [hunk 5]
```diff
@@ -94,4 +94,34 @@
...
+    padding-left: 22px;
+    background-color: rgb(0, 158, 231);
```
Comment (andy-armstrong) [inline done/static/css/done.css]: Keep colors out of the right-to-left class handling.

File: done/static/css/done.css (hunk 3 above)
Comment (andy-armstrong) [inline done/static/css/done.css]: http://anasnakawa.github.io/bi-app-sass/

File: done/static/css/done.css (hunk 4 above)
Comment (asadiqbal08) [inline done/static/css/done.css]: One thing that I want to highlight, There are pre-exist css/text in this file (you can look into done.css) that contain the 'text strings'. I just bind my css with `body.rtl` here. 
The comment same for other places.

File: done/static/css/done.css (hunk 3 above)
Comment (asadiqbal08) [inline done/static/css/done.css]: thanks, It seems me it based on .sass and we cannot use it in xblocks.

**Comment by andy-armstrong** [comment]
👍  looks great @asadiqbal08. 

It is worth noting that it won't really support RTL languages until the XBlock is able to translate its strings too.

**Comment by mattdrayer** [comment]
@pmitros can you please merge this PR and update the release tag, accordingly?

**Comment by pmitros** [comment]
@mattdrayer I merged the PR. It looks good. Thanks. I'm in an airplane right now, waiting to be cleared for takeoff. I'm not really confident in my ability to update release tag from here, so I gave you committer access. If you'd like to update the release tag, please go ahead. Otherwise, I could do it when I'm back in BOS.

**Comment by mattdrayer** [comment]
Thanks @pmitros!  I'll go ahead and add the release tag now -- have a safe flight!

**Comment by mattdrayer** [comment]
@pmitros looks like I don't have permission to push a release tag even with the committer access -- we can take care of this when you reconnect, however.

**Comment by pmitros** [comment]
I believe you need to accept the invite before you can push. It's showing
that it is not accepted.

**Comment by mattdrayer** [comment]
AH, of course -- the message was stuck in a junk mail filter :)

**Comment by mattdrayer** [comment]
@asadiqbal08 -- release tag is `release-2016-08-10`

**Pull request merged**